roles/openshift_examples/templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+EXAMPLES_BASE=$(pwd)/files/examples
```

Comment:
We could using `readlink -f` or some other tricks, but maybe it is not worth it. Maybe it is enough to document this script and usage in the REAME? (AFAICT it was just mentioned in the PRs description).